wolox-books/src/app/screens/register/register.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import patternPasswordValidation from 'src/app/constants/pattern-password-validation.constant';
+import IUser from 'src/app/interfaces/user.interface';
+import passwordConfirmation from 'src/app/validators/password-confirmation.validator';
+
+@Component({
+  selector: 'wb-register',
+  templateUrl: './register.component.html',
+  styleUrls: ['./register.component.scss']
+})
+export class RegisterComponent{
+
+  registerForm: FormGroup;
+
+  constructor(
+    private fb: FormBuilder,
+  ) {
+    console.log('passwordConfirmation :>> ', passwordConfirmation);
+    this.registerForm = fb.group({
+      'first_name': [ null ],
+      'last_name': [ null ],
```

Comment:
I think these two fields are required

Reply:
![image](https://user-images.githubusercontent.com/9323975/98860753-230dbe00-2432-11eb-850a-b7a696fb8b1d.png)

Reply:
The card don't mention that the 'first_name' or the 'last_name' are required

Reply:
I think all fields are required for the API to work 🤔

Reply:
OK! I will make all fields required ^^